dvc/system.py

```diff
@@ -26,7 +26,11 @@ def hardlink(source, link_name):
         return winlink.create(source, link_name)
 
     @staticmethod
-    def samefile(path1, path2):
+    def samefile(path1, path2, check_exist=False):
```

Comment:
check_exist is not used anywhere, unless i'm missing something. Though, checking if paths exist is very useful.

Reply:
Oh, it is used in find_cache, sorry.